platform-emf/src/main/java/com/softicar/platform/emf/management/importing/engine/EmfImportEngine.java

```diff
@@ -39,6 +39,7 @@ public EmfImportEngine(IEmfTable<R, P, S> table) {
 		ignoreActiveField();
 		ignoreGeneratedFields();
 		ignoreTransactionFields();
+		ignoreConcealedFields();
```

Comment:
i am not sure if we can ignore concealed fields here, can we talk about some examples?